inst/extdata/variable_details.csv

```diff
@@ -166,31 +171,39 @@ CCC_101,CCC_101_cat2_2,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p,
 CCC_101,CCC_101_cat2_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s","cchs2001_p::CCCA_101, cchs2003_p::CCCC_101,  cchs2005_p::CCCE_101, [CCC_101]",cat,NA::b,2,missing,missing,N/A,7:09,don't know (7); refusal (8); not stated (9),Diabetes,Do you have diabetes,
 CCC_101,CCC_101_cat2_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2009_s","cchs2001_p::CCCA_101, cchs2003_p::CCCC_101,  cchs2005_p::CCCE_101, [CCC_101]",cat,NA::b,2,else,else,N/A,else,else,Diabetes,Do you have diabetes,
-CCC_102,CCC_102_cat2_1,cat,cchs2009_s,,,,,,,,,,,,
-CCC_102,CCC_102_cat2_2,cat,cchs2009_s,,,,,,,,,,,,
-CCC_102,CCC_102_cat2_NA::a,cat,cchs2009_s,,,,,,,,,,,,
-CCC_102,CCC_102_cat2_NA::b,cat,cchs2009_s,,,,,,,,,,,,
-CCC_102,CCC_102_cat2_NA::b,cat,cchs2009_s,,,,,,,,,,,,
-CCC_10A,CCC_10A_cat2_1,cat,cchs2009_s,,,1,2,Yes diabetes diagnosed when pregnant,Yes diabetes diagnosed when pregnant,N/A,,,,,
-CCC_10A,CCC_10A_cat2_2,cat,cchs2009_s,,,2,2,No diabetes diagnosed when pregnant,No diabetes diagnosed when pregnant,N/A,,,,,
-CCC_10A,CCC_10A_cat2_NA::a,cat,cchs2009_s,,,NA::a,2,not applicable,not applicable,N/A,,,,,
-CCC_10A,CCC_10A_cat2_NA::b,cat,cchs2009_s,,,NA::b,2,missing,missing,N/A,,,,,
-CCC_10A,CCC_10A_cat2_NA::b,cat,cchs2009_s,,,NA::b,2,else,else,N/A,,,,,
-CCC_10B,CCC_10B_cat2_1,cat,cchs2009_s,,,1,2,Yes diabetes diagnosed other than pregnant,Yes diabetes diagnosed other than pregnant,N/A,,,,,
-CCC_10B,CCC_10B_cat2_2,cat,cchs2009_s,,,2,2,No diabetes diagnosed other than pregnant,No diabetes diagnosed other than pregnant,N/A,,,,,
-CCC_10B,CCC_10B_cat2_NA::a,cat,cchs2009_s,,,NA::a,2,not applicable,not applicable,N/A,,,,,
-CCC_10B,CCC_10B_cat2_NA::b,cat,cchs2009_s,,,NA::b,2,missing,missing,N/A,,,,,
-CCC_10B,CCC_10B_cat2_NA::b,cat,cchs2009_s,,,NA::b,2,else,else,N/A,,,,,
-CCC_10C,CCC_10C_cat2_1,cont,cchs2009_s,,,,,,,,,,,,
-CCC_10C,CCC_10C_cat2_2,cont,cchs2009_s,,,,,,,,,,,,
-CCC_10C,CCC_10C_cat2_NA::a,cont,cchs2009_s,,,,,,,,,,,,
-CCC_10C,CCC_10C_cat2_NA::b,cont,cchs2009_s,,,,,,,,,,,,
-CCC_10C,CCC_10C_cat2_NA::b,cont,cchs2009_s,,,,,,,,,,,,
-CCC_105,CCC_105_cat2_1,cat,cchs2009_s,,,1,2,Yes diabetes and currently taking insulin,Yes diabetes and currently taking insulin,N/A,,,,,
-CCC_105,CCC_105_cat2_2,cat,cchs2009_s,,,2,2,No diabetes and currently taking insulin,No diabetes and currently taking insulin,N/A,,,,,
-CCC_105,CCC_105_cat2_NA::a,cat,cchs2009_s,,,NA::a,2,not applicable,not applicable,N/A,,,,,
-CCC_105,CCC_105_cat2_NA::b,cat,cchs2009_s,,,NA::b,2,missing,missing,N/A,,,,,
-CCC_105,CCC_105_cat2_NA::b,cat,cchs2009_s,,,NA::b,2,else,else,N/A,,,,,
+CCC_102,CCC_102_cat2_1,cont,cchs2009_s,[CCC_102],cont,copy,N/A,Age diagnosed with diabetes,Age diagnosed with diabetes,years,,,,How old were you when this was first diagnosed?,

```

Comment:
Don't forget to add the range of values for the continuous variable (eg. 1:20 for values between 1 and 20). You can find this in the data dictionary.